MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -181,6 +213,49 @@ - (NSURLRequest *)requestForEvents:(NSArray *)events {
     return [request copy];
 }
 
+- (NSString *)mimeTypeForPath:(NSString *)path {
+    CFStringRef extension = (__bridge CFStringRef)[path pathExtension];
+    CFStringRef UTI = UTTypeCreatePreferredIdentifierForTag(kUTTagClassFilenameExtension, extension, NULL);
+    if (UTI == NULL) {
+        return nil;
+    }
+    
+    NSString *mimetype = CFBridgingRelease(UTTypeCopyPreferredTagWithClass(UTI, kUTTagClassMIMEType));
+    if (mimetype == NULL) {
```

Comment:
This `if` block isn't needed, since you return `mimetype` anyway. If it did return here, you'd also have an unreleased `UTI`.

Reply:
Good point